videoframes.py

```diff
@@ -209,6 +216,16 @@ def close_rewinded(self, event):
         # Let the garbage collector do its magic
         self.rewinded = None
 
+    def fullscreen_rewinded(self, _):
```

Comment:
Almost, rather it is our own function [call](https://github.com/kaszim/desktop-stream-viewer/blob/master/videoframes.py#L97)